src/video/video.js

```diff
@@ -136,7 +136,7 @@
             // sanitize potential given parameters
             settings.doubleBuffering = !!(settings.doubleBuffering);
             settings.autoScale = (settings.scale === "auto") || false;
-            settings.maintainAspectRatio = !!(settings.maintainAspectRatio);
+            settings.scaleMethod = !!(settings.scaleMethod);
```

Comment:
Shall I changed to?
settings.scaleMethod = (settings.scaleMethod === "fill" || settings.scaleMethod === "stretch") ? settings.scaleMethod : "default";